dipy/workflows/tests/test_workflow.py

```diff
@@ -0,0 +1,54 @@
+from nose.tools import assert_raises
+
+import os
+
+from nibabel.tmpdirs import TemporaryDirectory
+
+from dipy.data import get_data
+from dipy.workflows.segment import MedianOtsuFlow
+from dipy.workflows.workflow import Workflow
+
+
+def test_force_overwrite():
+    with TemporaryDirectory() as out_dir:
+        data_path, _, _ = get_data('small_25')
+        mo_flow = MedianOtsuFlow(output_strategy='absolute')
```

Comment:
OK - I understand. Yes - if you have several different possible inputs, you might want to check explicitly that the input is one of these, and raise an error for any other input.